extensions: Add Inspetkor Gadget (#151)
MicrosoftDocs/azure-management-docs: articles/azure-arc/kubernetes/extensions-release.md

For more information, see [What is Foundry Local on Azure Local?](/azure/azure-sovereign-clouds/private/foundry-local/what-is-foundry-local-on-azure-local).

## Inspektor Gadget (preview)

- **Supported distributions**: AKS

[Inspektor Gadget](https://inspektor-gadget.io/) is a framework that makes it easy to monitor, troubleshoot, and secure workloads running on Linux and Kubernetes. It consists of tools (*Gadgets*) that leverage [eBPF](https://ebpf.io/) programs. Their primary goal is to gather low-level kernel data to provide insights into specific system scenarios. The Inspektor Gadget framework manages the association of the collected data by using high-level references, such as Kubernetes resources. This integration ensures that a seamless connection exists between low-level insights and their corresponding high-level context. The integration streamlines the troubleshooting process and the collection of relevant information.

> For more information, see [Capture real-time system insights from an AKS cluster with Inspektor Gadget](/troubleshoot/azure/azure-kubernetes/logs/capture-system-insights-from-aks)

Reviewer: ```suggestion
For more information, see [What is the Inspektor Gadget extension for AKS?](/azure/aks/inspektor-gadget-overview)
```
